### ISSUE TITLE
Useless field tx_realurl_nocache 

### ISSUE THREAD

**Issue by rvock** [dmitryd/typo3-realurl#497]
Currently there is a field for the `pages` table called `tx_realurl_nocache`. It appears the following files

- ext_tables.sql
- Resources/Private/Language/locallang_db.xlf
- Configuration/TCA/Overrides/pages.php

The field does not appear in the backend and it appears to do nothing. So it's possible to remove the field. Alternatively it would be nice to make it work to disable cache for specific pages ;)

This commit introduced the field:
[0d1ff4](https://github.com/dmitryd/typo3-realurl/commit/0d1ff47449d566e268e86fae3942dc44d51a12cb)

**Comment by dmitryd** [comment]
Thanks!

**Comment by TrueType** [comment]
Hi @dmitryd 
I just had to struggle with a v1.12.8 installation of realurl on TYPO3 4.7 bloating the `tx_realurl_urlencodecache` table because of combinations of GET parameters. To handle it I fixed the backend display of the `tx_realurl_nocache` checkbox and could then switch on/off caching for a specific page. I am just wondering how this is done in the newer versions as I figure this is still superior to configure regular expressions? But I am just a newbie to realurl...
